unit-tests/py/test.py

```diff
@@ -58,6 +58,23 @@ def print_stack():
     for line in traceback.format_stack():
         print(line)
 
+# Function to check frame drops while streaming
+def check_frame_drops(frame, previous_frame, after_set_option = 0):
```

Comment:
I'd rather you passed the number of allowed drops in rather than a specific usage -- tomorrow it might be a different usage.

Reply:
Was this function part of Matan's unit-tests? If so, was it removed from there and moved here?